src/DocumentPicker.UWP/UwpStreamProviderImplementation.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using DocumentConverter.Plugin.Shared.StreamProvider;
+using Windows.Storage;
+
+namespace DocumentPicker.UWP
+{
+    public class UwpStreamProviderImplementation : ICustomStreamProvider
+    {
+        public async Task<Stream> OpenAsync(string filePath)
```

Comment:
please rename to `OpenReadAsync` so that it is clear, that this stream will not be writable!